tests/test_feature.rb

```diff
@@ -222,6 +224,28 @@ def test_feature_set_fabric
     assert(Feature.fabric_enabled?, "(#{fs}) is not enabled")
   end
 
+  def test_feature_set_fabricpath
+    if node.product_id[/N(3|8|9)/]
+      assert_nil(Feature.fabricpath_enabled?)
+      assert_raises(Cisco::UnsupportedError) { Feature.fabricpath_enable }
+      return
+    end
+    vdc_limit_f3_no_intf_needed(:set)
+    fs = 'feature-set fabricpath'
+    # Get current state of the feature-set
+    feature_set_installed = Feature.fabricpath_installed?
+
+    config("no #{fs} ; no install #{fs}") if feature_set_installed
```

Comment:
*nit* Why not just call `Feature.fabricpath_installed?` directly here?

Reply:
This test is just a copy/paste/rename from the other feature-set tests. I *think* the reason I used `config()` here back in the day was  because it's marginally faster to send both commands at once.

Reply:
I was not concerned about `config`.  I was suggesting you get rid of line 235 `feature_set_installed = Feature.fabricpath_installed?` and just do this.

```ruby
config("no #{fs} ; no install #{fs}") if Feature.fabricpath_installed?
```